modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,22 +1,21 @@
 // Module included in the following assemblies:
 //
-// * virt/install/virt-installing-virtctl.adoc
+// * virt/virt-using-the-cli-tools.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-installing-virtctl-client_{context}"]
-= Installing virtctl on Linux, Windows, and macOS
+= Installing the virtctl binary on {op-system-base} 9, Linux, Windows, and macOS
```

Comment:
```suggestion
= Installing the virtctl binary on {op-system-base} 9, Linux, Windows, or macOS
```
Not sure if this should be "or", since these are different options?